ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/persistence/PostgresIT.scala

```diff
@@ -17,9 +17,6 @@ import org.scalatest._
 import scala.concurrent.Await
 import scala.concurrent.duration.DurationInt
 
-import scala.concurrent.Await
```

Comment:
Weird, looks like you could simply remove these lines as the imports were unused but this should have caused a fatal warning. :thinking:

Reply:
I know right?